src/server/utils/validator.ts

```diff
@@ -90,10 +89,10 @@ export const checkAndReturnNFTSignaturePayload = <
   return updatedPayload;
 };
 
-export const isValidHttpUrl = (urlString: string): boolean => {
+export const isValidWebhookUrl = (input: string): boolean => {
   try {
-    const url = new URL(urlString);
-    return url.protocol === "http:" || url.protocol === "https:";
+    const url = new URL(input);
+    return url.protocol === "https:";
```

Comment:
removes `http:` as a valid protocol

Reply:
can we still allow 0.0.0.0/127.0.0.1 and localhost?

Reply:
Yeah I think this is reasonable.